contracts/SportMarkets/Parlay/ParlayMarketsAMM.sol

```diff
@@ -366,7 +371,7 @@ contract ParlayMarketsAMM is Initializable, ProxyOwned, ProxyPausable, ProxyReen
             _positions,
             _sUSDPaid
         );
-        uint safeBoxAmount = ((_sUSDPaid - sUSDAfterFees) * safeBoxImpact) / (safeBoxImpact + parlayAmmFee);
+        uint safeBoxAmount = _getSafeBoxAmount(_sUSDPaid, sUSDAfterFees, msg.sender);
```

Comment:
for future consistency and not braking the code:
`_getSafeBoxAmount(_sUSDPaid, sUSDAfterFees, _differentRecipient);`

with the audit merge
`_differentRecepient => _differentRecipient`